### PR TITLE
Add installation videos to the docs

### PR DESCRIPTION
**Pull request by JustinaPetr**
Add installation videos to the docs

**Proposed changes**:
Add installation videos to the docs

**Status (please check what you already did)**:
- [ ] added some tests for the functionality
- [x] updated the documentation
- [ ] updated the changelog (please check [changelog](https://github.com/RasaHQ/rasa/tree/master/changelog) for instructions)
- [ ] reformat files using `black` (please check [Readme](https://github.com/RasaHQ/rasa#code-style) for instructions)

### CODE REVIEW

**Comment by akelad** [comment]
@JustinaPetr I think I would rather have these videos be links we send people to in a "having trouble with the installation" section or similar rather than having them as embedded videos at the top. The reason for that is that if we have both videos and the written guide, it's kind of confusing for users which one they follow.
@edmacovaz do you have any additional thoughts? We discussed the videos on docs thing a while ago

**Comment by edmacovaz** [comment]
@akelad I tend to agree that it's best if we can keep getting started and install documentation looking simple. Is it worth us taking a moment to map out the different touchpoints that are being created at the moment (videos, docs/getting started, onboarding within Rasa X) to see what fits in where?

**Comment by akelad** [comment]
yeah that may be a good idea. The main question for me is: does the video have something the docs don't? If yes, we should complete the docs. If no, then i would list it as an alternative somewhere on the docs (i know some people prefer watching video, others prefer reading)

**Comment by JustinaPetr** [comment]
@akelad I see your point and I think it makes sense. My intention with adding it here was to give people an option to follow a video instead of reading if they prefer that (just like we have for the Rasa X docker-compose installation), but overall the content in the videos is just like the written one so I would be happy to move to a different place

**Comment by akelad** [comment]
I would say that we add it as a bullet point under line 25, saying something like "Prefer watching a video? Watch our installation series on youtube" and linking to the playlist. Thoughts?

**Comment by JustinaPetr** [comment]
@akelad I think that makes sense. I can make a change if everyone agrees to that

File: docs/user-guide/installation.rst [hunk 1]
```diff
@@ -42,6 +43,7 @@ When you're done installing, you can head over to the tutorial!
 Step-by-step Installation Guide
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
+
```
Comment (akelad) [inline docs/user-guide/installation.rst:46]: please remove that newline :D

**Pull request merged**